pulpito/controllers/nodes.py

```diff
@@ -69,7 +69,7 @@ def get_node(self, page=None):
         self.get_node_jobs(page=page)
         return self.node
 
-    def get_node_jobs(self, count=5, page=None):
+    def get_node_jobs(self, count=50, page=None):
```

Comment:
Sure, done. I just picked and arbitrarily larger number. It took me forever to dig back through and find the last passed job while investigating http://tracker.ceph.com/issues/19126